src/schemas.test.ts

```diff
@@ -0,0 +1,51 @@
+import { constants } from "ethers";
+
+import { recipientsFileSchema } from "./schemas";
+
+describe("schemas", () => {
+  describe("recipientsFileSchema", () => {
+    const valid = {
+      chainId: 1,
+      rewardToken: constants.AddressZero,
+      windowIndex: 0,
+      rewardsToDeposit: "10",
+      recipients: [
+        {
+          amount: "10",
+          account: constants.AddressZero,
```

Comment:
I just realized, one rule we should add is not allowing any address to be AddressZero, this would amount to burning ACX since no one owns 0x0